sample/sample9-entity-listeners/entity/Post.ts

```diff
@@ -49,7 +51,7 @@ export class Post {
         console.log(
             `event: Post "${this.title}" entity has been loaded and callback executed`,
         )
-        this.uid = Math.ceil(Math.random() * 1000)
+        this.uid = crypto.randomInt(0, 1000)
```

Comment:
It doesn't matter either way, but this sample code doesn't need to have a cryptographically secure random number.

Reply:
Yeah, it's just so we can get rid of the CodeQL warning.